### ISSUE TITLE
Problem in Lizard 1.10.3 installed via pip

### ISSUE THREAD

**Issue by rakhimov** [terryyin/lizard#94]
Lizard (installed via pip) is not reporting anything but the total line counts.
An example run can be seen [here](https://travis-ci.org/rakhimov/lizard/jobs/108168684).

**Comment by terryyin** [comment]
I created another release 1.10.4 and the problem is gone now. The scary thing is I don't understand why. There's very little difference in the code, which I don't think is the reason for the behaviour of 1.10.3. And I can only reproduce the problem of 1.10.3 on travis.

Now I suspect that it's because Lizard's ex-name Hfcca. I renamed hfcca to lizard but still keep hfcca alive and use one project to generate two packages. I still need to verify this idea.

In the future I will always use a clean clone to generate the pypi release. And probably give up hfcca maintenance.

Thank you very much.
